components/Dialogs/src/private/MDCAlertControllerView+Private.m

```diff
@@ -171,6 +171,11 @@ - (void)setTitleColor:(UIColor *)titleColor {
   _titleLabel.textColor = titleColor;
 }
 
+- (void)setTitleAlignment:(NSTextAlignment)titleAlignment {
+  _titleAlignment = titleAlignment;
+  _titleLabel.textAlignment = titleAlignment;
```

Comment:
Can we wire the titleAlignment getter/setter directly to _titleLabel.textAlignment and get rid of the _titleAligment ivar?